docs/spec/staking/state.md

```diff
@@ -1,204 +1,148 @@
+	BondedShares        sdk.Rat // sum of all shares distributed for the Bonded Pool
+	InflationLastTime   int64   // block which the last inflation was processed // TODO make time
+	Inflation           sdk.Rat // current annual inflation rate
+
+	DateLastCommissionReset int64  // unix timestamp for last commission accounting reset (daily)
 }
 
-type Description struct {
-    Name       string 
-    DateBonded string 
-    Identity   string 
-    Website    string 
-    Details    string 
+type PoolShares struct {
+	Status sdk.BondStatus // either: unbonded, unbonding, or bonded
+	Amount sdk.Rat        // total shares of type ShareKind
 }
 ```
 
-Candidate parameters are described:
-* Status: it can be Bonded (active validator), Unbonding (validator candidate) 
-  or Revoked
-* ConsensusPubKey: candidate public key that is used strictly for participating in 
-  consensus
-* GovernancePubKey: public key used by the validator for governance voting 
-* Owner: Address that is allowed to unbond coins.
-* GlobalStakeShares: Represents shares of `GlobalState.BondedPool` if 
-  `Candidate.Status` is `Bonded`; or shares of `GlobalState.Unbondingt Pool` 
-  otherwise
-* IssuedDelegatorShares: Sum of all shares a candidate issued to delegators 
-  (which includes the candidate's self-bond); a delegator share represents 
-  their stake in the Candidate's `GlobalStakeShares`
-* RedelegatingShares: The portion of `IssuedDelegatorShares` which are 
-  currently re-delegating to a new validator
-* VotingPower: Proportional to the amount of bonded tokens which the validator
-  has if `Candidate.Status` is `Bonded`; otherwise it is equal to `0`
-* Commission:  The commission rate of fees charged to any delegators
-* CommissionMax:  The maximum commission rate this candidate can charge each 
-  day from the date `GlobalState.DateLastCommissionReset` 
-* CommissionChangeRate: The maximum daily increase of the candidate commission
-* CommissionChangeToday: Counter for the amount of change to commission rate 
-  which has occurred today, reset on the first block of each day (UTC time)
-* ProposerRewardPool: reward pool for extra fees collected when this candidate
-  is the proposer of a block
-* Adjustment factor used to passively calculate each validators entitled fees
-  from `GlobalState.FeePool`
-* Description
-  * Name: moniker
-  * DateBonded: date determined which the validator was bonded
-  * Identity: optional field to provide a signature which verifies the 
-    validators identity (ex. UPort or Keybase)
-  * Website: optional website link
-  * Details: optional details
-
-### DelegatorBond
-
-Atom holders may delegate coins to candidates; under this circumstance their
-funds are held in a `DelegatorBond` data structure. It is owned by one 
-delegator, and is associated with the shares for one candidate. The sender of 
-the transaction is the owner of the bond.
+### Params
+ - index: n/a single-record
```

Comment:
What is this referring to? Removed by the parser?

Reply:
there is one record so we don't need an index because we're not sorting by anything